core/src/main/scala-2.12/cats/instances/all.scala

```diff
@@ -10,6 +10,7 @@ abstract class AllInstancesBinCompat
     with AllInstancesBinCompat4
     with AllInstancesBinCompat5
     with AllInstancesBinCompat6
+    with AllInstancesBinCompat7
```

Comment:
We shouldn't need any new `BinCompat` traits now that we've dropped 2.11. You should be able to add `PartialFunctionInstances directly to `AllInstances`, I believe?

Reply:
I'm not sure, getting this error when adding to `AllInstances`; I don't see anything obvious, might need some help

> [error]  * abstract synthetic method cats$instances$PartialFunctionInstances$_setter_$catsStdInstancesForPartialFunction_=(cats.arrow.ArrowChoice)Unit in interface cats.instances.PartialFunctionInstances is inherited by class AllInstances in current version.
[error]    filter with: ProblemFilters.exclude[InheritedNewAbstractMethodProblem]("cats.instances.AllInstances.cats$instances$PartialFunctionInstances$_setter_$catsStdInstancesForPartialFunction_=")
[error]  * abstract method catsStdInstancesForPartialFunction()cats.arrow.ArrowChoice in interface cats.instances.PartialFunctionInstances is inherited by class AllInstances in current version.
[error]    filter with: ProblemFilters.exclude[InheritedNewAbstractMethodProblem]("cats.instances.AllInstances.catsStdInstancesForPartialFunction")

Reply:
@gagandeepkalra Hmm, thanks, I'll take a look now.